docs/docs/explanations/architecture.mdx

```diff
@@ -0,0 +1,70 @@
+---
+id: infrastructure-architecture
+title: Nebari architecture and conceptual guide
```

Comment:
```suggestion
title: Nebari architecture
```

Reply:
Context: The documentation section where this page lives is called "Conceptual guides", so we needn't say it again explicitely :)